policies/recipes/recipedb.go

```diff
@@ -15,25 +15,69 @@
 package recipes
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"os"
 	"strconv"
 	"strings"
 )
 
+var (
+	dbPath = "~/placeholder_location/swr.json"
```

Comment:
FYI expansion of ~ is done by the shell and won't work here, you'll need to pick a real placeholder until we get the real real :)